index.css

```diff
@@ -10,15 +11,99 @@ body {
 
 li {
   text-decoration: none;
-  border-bottom: 1px solid;
-  padding-bottom: 8px;
+  padding: 8px;
 }
 
 input {
   display: block;
   margin-bottom: 12px;
+  padding: 12px;
+  border: 1px solid;
+}
+
+#book-entry {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-self: center;
+}
+
+#books {
+  border-width: 1px;
+  border-style: solid;
+}
```

Comment:
- [ ] Always use the class attribute for multiple elements (do not use ID selector for multiple elements). Kindly change this to class selectors. Please see the link for [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
@mikenath223 book-entry and #books are not multiple elements. I believe it according to the best practice